### ISSUE TITLE
Empty Test Lint

### ISSUE THREAD

**Issue by fhupp** [rust-lang/rust-clippy#17488]
### What it does

This lint should suggest removing empty test functions.

### Advantage

By applying this lint it would make source code cleaner / shorted by removing tests that do nothing. 

### Example

```rust
#[test]   
fn my_empty_test() {

}
```

Could be removed

### Comparison with existing lints

To my knowledge this lint doesn't exist yet.

**Comment by Jarcho** [comment]
This can be a more general lint for tests which can never fail or always fail. Empty tests aren't particularly unique other than being the degenerate case.
